src/main/java/Crystal.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Crystal {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        System.out.println("____________________________________________________________\n"
+                + "Hola! I'm Crystal.\n"
+                + "What can I do for you today?\n"
+                + "____________________________________________________________\n");
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
```

Comment:
Perhaps comments could be added for each if-else loop for other programmers to better understand the program flow?